packages/backend/src/services/UnfurlService/UnfurlService.ts

```diff
@@ -505,12 +506,25 @@ export class UnfurlService extends BaseService {
                                         },
                                     ); // NOTE: No await here
                                 });
-                            // We wait for the sql charts to load
+                            // We wait for the sql charts to load and for the query to finish
+                            // NOTE: We can't combine these promises into a single otherwise Playwright will not capture the response
                             const sqlChartResultsPromises =
                                 sqlChartTileUuids?.map(
                                     (id) =>
                                         page?.waitForResponse(
-                                            /\/sqlRunner\/results/,
+                                            new RegExp(
+                                                `/sqlRunner/saved/${id}`,
+                                            ),
+                                            {
+                                                timeout: 60000,
+                                            },
+                                        ), // NOTE: No await here
+                                );
```

Comment:
waits for 2 response regexes
- chart results (like name, config, etc)
- chart query results (against the warehouse)

Note: we can't combine these 2 bc playwright won't await for them properly